include/mqtt/property.hpp

```diff
@@ -915,123 +559,51 @@ class user_property_ref {
     std::size_t num_of_const_buffer_sequence() const {
```

Comment:
Can't this function be static constexpr?

Reply:
Yes, it should be. I will fix it. Including other three member functions.